### ISSUE TITLE
Tests for twisted.internet should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7866]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7866|
|Type|enhancement|
|Created|2015-04-23 15:40:40Z|
|Branch|https://github.com/twisted/twisted/tree/test-names-t.i-7866|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [misc_7866.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/04b/04b43b61546117a817533ba5fc09009bc06f16f4/a9e8357f15b38c6e614c3cf78955d6173602a1a3.patch) (15613 bytes) - added by eeshangarg on 2015-04-23 16:05:29Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7866 7866
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_test_names_t_i_7866 branches/test-names-t.i-7866
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1429803640056586 1429803640056586
changetime__1429876223482968 1429876223482968
version__None None
owner__eeshangarg eeshangarg
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg removed owner|
|-|-|

Hello!

I just added a patch for this (misc_7866.patch​). As a part of this patch, I also made a small change to twisted.internet.test.reactormixins.ReactorBuilder.makeTestCaseClasses <https://github.com/twisted/twisted/blob/trunk/twisted/internet/test/reactormixins.py#L327>, so that names of test classes generated by this method conform to the new style of naming test classes. :-)

Regards,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [44536]) Branching to test-names-t.i-7866.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @eeshangarg|
|-|-|

Changes looks good. Thanks!

Waiting for results from buildbot and will merge.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

(In [44551]) Merge test-names-t.i-7866: Update test in twisted.internet to conform to the new style of naming test classes.

Author: eeshangarg
Reviewers: adiroiban
Fixes: [#7866](https://github.com/twisted/twisted/issues/7866)
